mods/ts/rules/defaults.yaml

```diff
@@ -566,6 +569,9 @@
 	Carryable:
 		RequiresCondition: !inside-tunnel
 	RevealOnFire:
+	RejectsOrders@tunnel:
+		Reject: Stop
```

Comment:
Right, well that makes this only slightly less trivial: https://github.com/OpenRA/OpenRA/commit/2e8f8f8d08daf698137666095bae1af5e066e322?w=1